npm/private/npm_package_store.bzl

```diff
@@ -364,9 +362,16 @@ deps of npm_package_store must be in the same package.""" % (ctx.label.package,
             for npm_package_store_info in npm_package_store_infos
         ])
 
+    files_depset = depset(files)
+
     providers = [
+        js_info(
```

Comment:
Thank you for simplifying the diff so it is 4 lines added instead of 50 changed 😅 👍